components/FilteredList.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import Card from "@/components/Card";
+
+export default function FilteredList({ tasks, positionCheck }) {
+  const filteredTasks = tasks.filter((task) => task.position === positionCheck);
+  return (
+    <StyledUl>
+      {filteredTasks.map((mappedTask) => (
+        <StyledLi key={mappedTask.id}>
+          <Card
+            name={mappedTask.name}
+            note={mappedTask.note}
+            time={mappedTask.time}
+          />
+        </StyledLi>
+      ))}
+    </StyledUl>
+  );
+}
```

Comment:
Yes I could, but I used "task" and "tasks" a lot already and I noticed people get confused because of it. So I tried to describe it more, because in the end a human has to understand the code. :D